projects/lkft-android/fastboot.jinja2

```diff
@@ -0,0 +1,119 @@
+{% extends "include/fastboot.jinja2" %}
+
+{% set deploy_download_timeout = deploy_download_timeout|default(15) %}
+{% set deploy_fastboot_timeout = deploy_fastboot_timeout|default(10) %}
+{% set pre_boot_timeout = pre_boot_timeout|default(0) %}
+{% set post_boot_timeout = post_boot_timeout|default(10) %}
+
+{# target_deploy_timeout is used to calculate the job_timeout defined in the root master.jinja2 file #}
```

Comment:
Just want to mention here, the following lines on the target_deploy_timeout will be deleted after the PR here merged: https://github.com/Linaro/lava-test-plans/pull/187/files

Reply:
awesome